src/components/template-tab.js

```diff
@@ -59,36 +76,20 @@ class TemplateTab extends React.Component {
         }
       })
       .otherwise((err)=>{
-        let errorMessage = err.error;
-
-        if(err.status && err.status.text){
-          errorMessage = err.status.text;
-        }
-
-        this.setState({
-          errorMessage: errorMessage
-        });
+        this.handleError(err);
```

Comment:
since you've already used .bind on this.handleError, you can just pass it directly to the .otherwise, no need for another function wrapper